content/blog/survey-02/index.md

```diff
@@ -0,0 +1,221 @@
+- Make the newsletter more consistent in timing and quantity
+- Have more editors to not overburden the current ones
+- Have some more clarity of purpose
+
+
+## Comments
+
+![Comments wordcloud](comment.png)
+
+This last free-text question was meant for any additional comments readers might have. 
+The word cloud above is dominated by one sentence: "Thank you for your work". Thank you very very much for your kind words! 
+We are working on this newsletter in our free time because we love the community and Rust game development, so reading this means a lot to us.
+
+## Correlations
+
+We were interested in how the responses to some questions correlated with how much people had already contributed to the newsletter.
+Long story short: it seems like there is no significant correlation between how much people contributed and how they answered the other questions.
+
+Let's look at the correlations in turn now.
+Note that all the following plots are jittered to make the data more readable.
+
+
+### Excitement By Contributions
+
+![Excitement correlation](excitement_contributions.png)
+
+We hypothesized that people who contributed more to the newsletter would be more excited about it. 
+We found no evidence for this (the p-value of a Jonckheere-Terpstra test for increasing trend is 0.986). 
+Based on the plot above, we then hypothesized that the opposite might be true, namely that frequent contributors are less excited about the newsletter.
+This actually might be the case (p-value is 0.021), but do not take this as a strong result.
+It is a posthoc hypothesis and the resulting p-value is not very low considering the number of tests we run in this analysis. 
+For these reasons, we do not consider this result to be significant.
+
+### Feelings About AI By Contributions
+
+![AI correlation](ai_contributions.png)
+
+We hypothesized that there would be a correlation between how much people contributed to the newsletter and how they felt about AI-generated summaries. 
+We found no evidence for this (the p-value of a Jonckheere-Terpstra test for a two-sided alternative is 0.57).
+
+### Ease of Contributing By Contributions
+
+![Ease correlation](ease_contributions.png)
+
+We hypothesized that there would be a correlation between how much people contributed to the newsletter and how easily they found it to contribute.
+
+We found no evidence for this (the p-value of a Jonckheere-Terpstra test for a two-sided alternative is 0.25).
+
+## Conclusion
+
+All in all, we are happy with the results of the survey. 
+It seems like our readers are generally happy with the newsletter, and have good ideas on how to improve it. 
+We will discuss how to implement these ideas in the future and keep you updated on our progress.
+
+If you are interested in helping us out, we are always looking for new editors and contributors. Just leave us a message on [Discord] or [GitHub].
+
+Again, thank you very much for your feedback. Rebooting the newsletter was a big ordeal for us,
```

Comment:
`ordeal`, or did you mean `deal`?

Just making sure!

Reply:
both :)